documentation/home.md

```diff
@@ -0,0 +1,30 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector helps developers design, build and manage AI agents within Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.
+
+## Key Features
+
+The MuleSoft AI Chain Connector simplifies AI integration into MuleSoft applications with:
+
+- Seamless Interaction with LLMs: Effortlessly integrate and utilize LLMs for tasks such as natural language processing, text generation, text classification and more complex language tasks.
+- Embeddings and Search: Leverage embeddings to handle tasks like text similarity, document search, and clustering, all within MuleSoft applications.
+- Optimized Performance: Designed for high efficiency and performance in enterprise-grade MuleSoft applications, ensuring smooth handling of AI operations.
+- File-Based Local Vector Stores: Create simple, file-based vector stores directly within the connector, ideal for local use or POC designs. 
+- Comprehensive AI Tools and Services: Access a wide array of AI-driven features, including Retrieval-Augmented Generation (RAG) for document retrieval, dynamic tool integration (Function Calling), and image model support for tasks like recognition and manipulation.
+
+## Integration with the Anypoint Platform Ecosystem
+
+The MuleSoft AI Chain Connector integrates tightly with the Anypoint Platform ecosystem, offering:
+
+- Customizable Tooling: Use configuration files for flexible setups tailored to your needs. 
+- Easy Tooling Management: Access and manage a wide range of AI tools directly through Anypoint Exchange.
+
+- AI Lifecycle Management: Manage the entire lifecycle of AI agents, from design to deployment, using Anypoint Platform.
+- Centralized AI Design: Design agents for seamless workflow integration.
+- Comprehensive Monitoring: Track and monitor your AI agents with Anypoint Monitoring and visualization tools.
+- Low-Code Development: Build AI applications with minimal code using Anypoint Studio and Anypoint Code Builder.
+- Built-In Testing: Ensure reliability with a robust unit-testing framework using MUnit in Anypoint Studio.
+
+## See Also
+
+- Add links to product doc
```

Comment:
@valkyrie69 can you provide a link that @dipeshkumardutta-sfemu can add to link from Exchange to docs.mulesoft.com?